specification/eventhub/resource-manager/Microsoft.EventHub/preview/2024-05-01-preview/namespaces.json

```diff
@@ -382,7 +382,7 @@
```

Comment:
This looks same as common-types v5 version. Why not update everywhere to refer common types instead of having a copy in eventhub ?

Reply:
Hey @ramoka178, I will take care of this in next release. currently it is blocking the SDK release.
can you please approve.